lib/IRGen/GenStruct.cpp

```diff
@@ -680,6 +720,12 @@ class ClangRecordLowering {
 
   const TypeInfo *createTypeInfo(llvm::StructType *llvmType) {
     llvmType->setBody(LLVMFields, /*packed*/ true);
+    if (SwiftType.getStructOrBoundGenericStruct()
+            ->isCxxNotTriviallyCopyable()) {
+      return ClangRecordAddressOnlyTypeInfo::create(
+          FieldInfos, llvmType, TotalStride, std::move(SpareBits),
```

Comment:
The `FieldInfos` vector will contain private data members that we're not importing. It would be good to skip private data members in `ClangRecordLowering` (in a future PR).

Reply:
I can make a follow up PR for this.

Reply:
SGTM!